packages/edge/scripts/fix-links.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+const path = require('path');
+
+const docs = path.join(__dirname, '..', 'docs');
+const interfaces = path.join(docs, 'interfaces');
+
+for (const dir of [docs, interfaces]) {
```

Comment:
**suggestion (non-blocking):** add comment about why this is necessary

Can we add a comment here about what exactly is happening and why? It could link to the issue you created: https://github.com/TypeStrong/typedoc/issues/2140